Project.toml

```diff
@@ -6,12 +6,13 @@ version = "0.1.1"
 DebuggerFramework = "67417a49-6d77-5db2-98c7-c13144130cd2"
 Markdown = "d6f4376e-aef5-505a-96c1-9c027394607a"
 REPL = "3fa0cd96-eef1-5676-8a61-b3b8758bbffb"
-TerminalRegressionTests = "98bfdc55-cc95-5876-a49a-74609291cbe0"
 
 [extras]
-VT100 = "7774df62-37c0-5c21-b34d-f6d7f98f54bc"
 TerminalRegressionTests = "98bfdc55-cc95-5876-a49a-74609291cbe0"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
+VT100 = "7774df62-37c0-5c21-b34d-f6d7f98f54bc"
+InteractiveUtils = "b77e0a4c-d291-57a0-90e8-8db25a27a240"
 
 [targets]
 test = ["Test", "TerminalRegressionTests", "VT100"]
+build = ["InteractiveUtils"]
```

Comment:
There is currently no `build` target.

Reply:
Recommended approach? Just make it a dependency of the package?

Reply:
Yes